digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/GraphPartitioner.scala

```diff
@@ -8,8 +8,8 @@ import org.geotools.graph.structure.basic.BasicEdge
 import scala.collection.JavaConversions._
 
 trait GraphPartitioner {
-  protected def clusterLinks[T <: PolyLine](links: Seq[T]): Seq[Graph] = {
-    val generator = new BasicLineGraphGenerator(0.5)
+  protected def clusterLinks[T <: PolyLine](links: Seq[T], tolerance: Double = 0.5): Seq[Graph] = {
```

Comment:
should we use fi.liikennevirasto.viite.MaxSuravageToleranceToGeometry for tolerance value?

Reply:
Partitioner can use any given value, but for viite the "connectedness" is defined as < 0.1 meters. (It's used in projectlinkpartitioner, for example)